src/lib/__tests__/date.test.ts

```diff
@@ -1,51 +1,271 @@
 import { isNull } from "lodash"
 import moment, { Moment } from "moment"
-import { exhibitionPeriod, exhibitionStatus } from "lib/date"
+import {
+  dateRange,
+  exhibitionStatus,
+  datesWithHours,
+  datesWithDaysAndHours,
+  singleDateWithoutHour,
+  singleDate,
+  singleHour,
+  formattedOpeningHours,
+  datesAreSameDay,
+  formattedHours,
+} from "lib/date"
 
-describe("date", () => {
-  describe("exhibitionPeriod", () => {
-    it("includes the start and end date", () => {
-      const period = exhibitionPeriod(
-        moment("2011-01-01T00:00-0400"),
-        moment("2014-04-19T00:00-0400")
+describe("date formatting", () => {
+  describe("formattedHours", () => {
+    it("If within same am/pm", () => {
+      const period = formattedHours(
+        "2022-12-30T08:00:00+00:00",
+        "2022-12-30T10:00:00+00:00"
+      )
+      expect(period).toBe("8:00 – 10:00am")
```

Comment:
This is missing the TZ name, as per [the spec](https://www.notion.so/artsy/Formatting-Time-f6f4aaeeba414583a207013fec5a9794#16c45f5d8e514e6db248990d97cd7d84) and this should also take a destination TZ for what TZ to consider these times in.

Same applies to test below.